src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let currentTab = tabs.find(tab => tab.id === selectedTabId);
+
+  if (!currentTab) {
+    const defaultValue = 0;
+
+    currentTab = tabs[defaultValue];
+  }
+
+  const doIfChanged = (tab) => {
```

Comment:
```suggestion
  const handleClick = (tab) => {
```
use such names for handlers